### ISSUE TITLE
configuration will not be applied

### ISSUE THREAD

**Issue by equi2k** [Louisvdw/dbus-serialbattery#790]
### Describe the problem

Hello all,
I have a JBD BMS via RS485 on the Raspberry with Venus OS and I can't adjust the max charge voltage.

I have set the following:

config.ini
MAX_CELL_VOLTAGE = 3.500

And to test again in the config.default.ini:
MIN_CELL_VOLTAGE = 2.900
MAX_CELL_VOLTAGE = 3.500
FLOAT_CELL_VOLTAGE = 3.375

But this has no effect on the max charge voltage, it is always set to 55.2V.

Can someone help me?

Regards, Christoph

### Driver version

1.0.20230531

### Venus OS device type

Raspberry Pi

### Venus OS version

2.93

### BMS type

Smart BMS (LLT, JBD, Overkill Solar)

### Cell count

16

### Connection type

Serial USB adapter to RS485

### Config file

```ini
[DEFAULT]

; If you want to add custom values/settings, then check the values/settings you want to change in "config.default.ini"
; and insert them below to persist future driver updates.

; Example (remove the semicolon ";" to uncomment and activate the value/setting):
; MAX_BATTERY_CHARGE_CURRENT = 50.0
; MAX_BATTERY_DISCHARGE_CURRENT = 60.0
MAX_CELL_VOLTAGE = 3.500
```

### Relevant log output

```shell
No permission...
```

**Comment by mr-manuel** [comment]
Did you reinstall the driver after changing the settings?

```bash
/data/etc/dbus-serialbattery/reinstall-local.sh
```

Please post the output of:

```bash
ls  -l /data/etc/dbus-serialbattery

cat /data/etc/dbus-serialbattery/config.ini

cat /opt/victronenergy/dbus-serialbattery/config.ini
```

**Comment by mr-manuel** [comment]
@equi2k any updates?

**Comment by nihonjin98** [comment]
I am having the same problem.  I tried to edit using winscp but it wouldnt keep the changes after a reboot.  I then used putty and again wouldnt hold the changes.  I ran the commands you posted above and they only showed the default config file without any of the changes i made.  I need to inverter the charge current to -1 and want to change the default charge and discharge voltage.

How do I go about changing the default config.ini and not loosing the changes after a reboot?  This is what i want to have int he file:

MAX_BATTERY_CHARGE_CURRENT = 125.0
MAX_BATTERY_DISCHARGE_CURRENT = 250.0
INVERT_CURRENT_MEASUREMENT = -1
